.github/workflows/release-docs.yml

```
@@ -1,5 +1,9 @@
 name: Asgardeo Docs Release
 
+concurrency:
+  group: pr-builder-${{ github.event.pull_request.number }}
```

Let's make the group name constant, so that all release jobs fall under one group. then cancel-in-progress will cancel all running jobs in the groups except one.

```suggestion
  group: doc-release
```